src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,7 +19,7 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.FullName;
+            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
```

Comment:
Yeah, I got that behaviour too. We should probably be checking the repo where the file is and not the current repo to determine the link to go to. That's likely a bigger change though :P